packages/govuk-frontend/src/govuk/components/pagination/pagination.yaml

```
@@ -36,7 +36,11 @@ params:
       - name: text
         type: string
         required: false
-        description: The link text to the previous page. Defaults to 'Previous'.
+        description: The text content of the link to the previous page. Defaults to 'Previous page', with 'page' being visually hidden. If `previous.html` is supplied, this is ignored.
```

Sorry, similar comment to https://github.com/alphagov/govuk-frontend/pull/3902/files#r1255539276

Shall we stick to _provided_ wording in both fields, rather than _supplied_?

```suggestion
        description: The text content of the link to the previous page. Defaults to 'Previous page', with 'page' being visually hidden. If `html` is provided, the `text` option will be ignored.
```